cypress.config.js

```diff
@@ -79,7 +79,7 @@ module.exports = defineConfig({
 		supportFile: 'tests/cypress/support/index.js',
 		testIsolation: false,
 		excludeSpecPattern: [
-			"tests/cypress/integration/z-newfold-labs/wp-module-onboarding/steps/design/**",
+			"tests/cypress/integration/z-newfold-labs/wp-module-onboarding/2-intermediate-steps/4-design/**",
```

Comment:
Are the design step test cases not working?

Reply:
No, not yet working